Database/Corrections/QuestieQuestFixes.lua

```diff
@@ -230,6 +231,17 @@ function QuestieQuestFixes:Load()
             [QuestieDB.questKeys.sourceItemId] = 18746,
             [QuestieDB.questKeys.zoneOrSort] = 1637,
         },
+        [7669] = { --#1449
+            [QuestieDB.questKeys.name] = "Again Into the Great Ossuary",
+            [QuestieDB.questKeys.startedBy] = {{928,},nil,nil,},            -- Quest is started by Lord Grayson Shadowbreaker
```

Comment:
[Lord Grayson Shadowbreaker](https://classic.wowhead.com/npc=928/lord-grayson-shadowbreaker) is an Alliance NPC and therefore it doesn't make much sense for him to start a Shaman quest 🤔

Reply:
I updated the start/end NPC.
Source: https://www.wow-freakz.com/quest_helper.php?quest=7669&name=again-into-the-great-ossuary